src/webviews/src/modules/oauth/oauth-create-free-db/OAuthCreateFreeDb.spec.tsx

```diff
@@ -35,4 +37,21 @@ describe('OAuthConnectFreeDb', () => {
       },
     })
   })
+
+  it('should open page and oauth sso modal compressed button is clicked', () => {
+    const { queryByTestId } = render(<OAuthCreateFreeDb source={OAuthSocialSource.DatabasesList} compressed={true}/>)
+
+    const compressedCreateBtn = queryByTestId('create-free-db-btn')
+    expect(compressedCreateBtn).toBeInTheDocument()
+
+    fireEvent.click(compressedCreateBtn as HTMLButtonElement)
+
+    expect(vscodeApi.postMessage).toBeCalledWith({
+      action: VscodeMessageAction.OpenAddDatabase,
+      data: {
+        ssoFlow: OAuthSocialAction.Create,
+        source: OAuthSocialSource.DatabasesList,
+      },
+    })
+  })
```

Comment:
Consider the following scenario for an additional test or tests - no `ssoFlow` or `source`.  I'm not sure what is the best suite for such a test tho, but IMO we should check for it.